### PR TITLE
cni: split DefaultCniImage to cni based images

### PR DESCRIPTION
**Pull request by nirmoy**
## What does this PR change?

split DefaultCniImage to DefaultFlannelImage and DefaultCiliumImage to ease configuration 
( Previous behaviour, Current behaviour etc).

## Documentation
- No documentation needed

### CODE REVIEW

File: pkg/cni/flannel/flannel.go [hunk 1]
```diff
@@ -142,6 +142,11 @@ func EnsureFlannelAddon(cfg *config.KubicInitConfiguration, client clientset.Int
 		return fmt.Errorf("error when parsing flannel configmap template: %v", err)
 	}
 
+	image := cfg.Network.Cni.Image
+	if len(image) == 0 {
+		image = config.DefaultFlannelImage
+	}
+	fmt.Printf("* %s %s ", image, config.DefaultFlannelImage)
```
Comment (MalloZup) [inline pkg/cni/flannel/flannel.go]: remove this line or use the logger

Comment (nirmoy) [reply pkg/cni/flannel/flannel.go]: @MalloZup  oops sorry about that, it was my debug :/ Resolved

Comment (MalloZup) [reply pkg/cni/flannel/flannel.go]: @nirmoy yop no problem i imagined it  looks good :+1: congrats

File: pkg/cni/cilium/cilium.go [hunk 2]
```diff
@@ -206,6 +206,11 @@ func EnsureCiliumAddon(cfg *config.KubicInitConfiguration, client clientset.Inte
 		return fmt.Errorf("error when creating cilium secret  %v", err)
 	}
 
+	image := cfg.Network.Cni.Image
+	if len(image) == 0 {
```
[the remark this answers is not in the document]
Comment (inercia) [reply pkg/cni/cilium/cilium.go]: I personally think it is a question of taste... :thinking:

**Comment by MalloZup** [comment]
Thx @nirmoy

**Pull request merged**